setup.py

```diff
@@ -10,9 +10,9 @@
 # NOTE: These are tested in `continuous_integration/test_imports.sh` If
 # you modify these, make sure to change the corresponding line there.
 extras_require = {
-    "array": ["numpy >= 1.16"],
+    "array": ["numpy >= 1.17"],
```

Comment:
From the discussion starting at https://github.com/dask/dask/issues/7378#issuecomment-880229623 and [this table](https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table), I was under the impression we were bumping the minimum NumPy version to 1.18, not 1.17. Though I could be missing something

Reply:
Ha! You're right. Clearly haven't had enough coffee 😅 ☕ Will update momentarily

Reply:
Ok fixed now. Thanks for catching that 😄